### PR TITLE
tests: header size limitation, closes #740

### LINKED ISSUE #740: Test header size limitation
Headers are limited since https://github.com/nspcc-dev/neofs-node/pull/2749 (see https://github.com/nspcc-dev/neofs-api/issues/262 also), try creating an object with big header (lots of attributes) and it should fail.

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: pytest_tests/tests/object/test_object_api.py [hunk 1]
```diff
@@ -736,6 +737,25 @@ def test_object_get_range_hash_negatives(
...
                         )
+                        
+    def test_put_object_header_limitation(
+        self, default_wallet: NodeWallet, container: str, simple_object_size: int
+    ):
+        file_path = generate_file(simple_object_size)
+        attr_key = "attr"
+        attr_val = "val"
+        attr_len = len(attr_key) + len(attr_val) + 9
+        api_limit = 16384
```
Comment (cthulhu-rider) [inline pytest_tests/tests/object/test_object_api.py:748]: this is a global sys const, suggest to declare it in appropriate upper level and add ref https://github.com/nspcc-dev/neofs-api/blob/5c3535423c564fe63991812cb84d8334db1c553a/object/service.proto#L319

File: pytest_tests/tests/object/test_object_api.py [hunk 2]
```diff
@@ -736,6 +737,25 @@ def test_object_get_range_hash_negatives(
...
                             range_cut=range_cut,
                         )
+                        
+    def test_put_object_header_limitation(
+        self, default_wallet: NodeWallet, container: str, simple_object_size: int
+    ):
+        file_path = generate_file(simple_object_size)
+        attr_key = "attr"
+        attr_val = "val"
+        attr_len = len(attr_key) + len(attr_val) + 9
```
Comment (cthulhu-rider) [inline pytest_tests/tests/object/test_object_api.py:747]: what `9` stands for?

File: pytest_tests/tests/object/test_object_api.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/tests/object/test_object_api.py]: Fixed

File: pytest_tests/tests/object/test_object_api.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/tests/object/test_object_api.py]: It was the "accurately" picked hardcoded number with the great sense in it :) I updated this.

**Review by roman-khimov — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**